app/views/podcast/feed.rss.builder

```diff
@@ -0,0 +1,92 @@
+xml.instruct! :xml, version: "1.0", encoding: "UTF-8"
+xml.rss "version"      => "2.0",
+        "xmlns:atom"   => "http://www.w3.org/2005/Atom",
+        "xmlns:cc"     => "http://web.resource.org/cc/",
+        "xmlns:itunes" => "http://www.itunes.com/dtds/podcast-1.0.dtd",
+        "xmlns:media"  => "http://search.yahoo.com/mrss/",
+        "xmlns:rdf"    => "http://www.w3.org/1999/02/22-rdf-syntax-ns#" do
+
+  xml.channel do
+    xml.tag!("atom:link", href: podcast_feed_url, rel: "self", type: "application/rss+xml")
+    xml.title @podcast.title
+    xml.pubDate @episodes.first.published_at.to_s(:rfc822)
+    xml.lastBuildDate @episodes.first.published_at.to_s(:rfc822)
+    xml.link podcast_url
+    xml.language @podcast.language
+    xml.copyright do
+      xml.cdata! @podcast.copyright
+    end
+    xml.docs podcast_url
+    xml.managingEditor "#{@podcast.itunes_owner_email} (#{@podcast.itunes_owner_email})"
+    xml.tag!("itunes:summary") do
+      xml.cdata! @podcast.itunes_summary
+    end
+    xml.image do
+      xml.url @podcast.image
+      xml.title @podcast.title
+      xml.link do
+        xml.cdata! podcast_url
+      end
+    end
+    xml.tag!("itunes:author") do
+      xml.text! @podcast.itunes_author
+    end
+    xml.tag!("itunes:keywords") do
+      xml.text! @podcast.tags
+    end
+
+    # TODO: These are hardcoded for now.  Find a simple way to nest categories.
+    xml.tag!("itunes:category", text: "News &amp; Politics")
```

Comment:
Looks like `builder` is html encoding the `text:` string. 

This
`"News &amp; Politics"`

is becoming this:
`"News &amp;amp; Politics"`

I think you can just use `"News & Politics"`